eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/views/EventFilterView.java

```diff
@@ -112,6 +113,17 @@ private void init(Iterable<CategoryOption> categoryOptions)
         // Populate the drop-down
         mDropDown.removeAllViews();
         LayoutInflater inflater = LayoutInflater.from(getContext());
+
+        inflater.inflate(R.layout.schedjoules_view_divider, mDropDown, true);
+
+        // Add show all item
+        SchedjoulesViewFilterItemBinding clearItem = DataBindingUtil.inflate(inflater, R.layout.schedjoules_view_filter_item, mDropDown, true);
+        clearItem.schedjoulesFilterItemLabel.setText(R.string.schedjoules_filter_show_all);
+        clearItem.getRoot().setOnClickListener(new ShowAllSelectListener());
+        clearItem.getRoot().setFocusable(true);
```

Comment:
You need to make the items focusable which are supposed to be focusable. Making each item focusable allows to navigate from one item to the next one with up and down arrow key.
Not sure what you mean with the outside touch handling in this context.